integreat_cms/cms/views/events/event_form_view.py

```diff
@@ -236,6 +236,13 @@ def post(self, request, **kwargs):
                     language__in=languages
                 ).update(status=status.DRAFT)
 
+            if (
+                event_translation_form.instance.status == status.PUBLIC
+                and event_translation_form.instance.minor_edit
+            ):
+                event_translation_form.instance.poi.translations.filter(
```

Comment:
```suggestion
                event_translation_form.instance.event.translations.filter(
```

Reply:
It's a bit sad that our tests did not reveal this error. Unfortunately we can't add automated tests for every edge case of the form submissions... at least for the future, we could test this specific case by adding
```
            (
                "edit_event",
                PRIV_STAFF_ROLES + REGION_ROLES,
                {
                    "title": "new title",
                    "start_date": "2030-01-01",
                    "end_date": "2030-01-01",
                    "is_all_day": True,
                    "status": status.PUBLIC,
                    "minor_edit": True
                },
            ),
```
to the [view test config](https://github.com/digitalfabrik/integreat-cms/blob/3ed0649012dfa9945ea183e35ae401ef0a950413/tests/cms/views/view_config.py#L736-L746).